src/renderer/initializers/kube-object-detail-registry.tsx

```diff
@@ -64,393 +61,395 @@ export function intiKubeObjectDetailRegistry() {
         kind: "HorizontalPodAutoscaler",
         apiVersions: ["autoscaling/v2beta1"],
         components: {
-          Details: (props: KubeObjectDetailsProps<endpoints.HorizontalPodAutoscaler>) => <HpaDetails {...props}/>,
+          // Note: this line is left in the long form as a validation that this usecase is valid
+          Details: (props: HpaDetailsProps) => <HpaDetails {...props}/>,
         }
       },
       {
         kind: "HorizontalPodAutoscaler",
         apiVersions: ["autoscaling/v2beta1"],
         priority: 5,
         components: {
-          Details: (props: KubeObjectDetailsProps<KubeObject>) => <KubeEventDetails {...props}/>,
-        }
+          // Note: this line is left in the long form as a validation that this usecase is valid
+          Details: (props: KubeObjectDetailsProps) => <KubeEventDetails {...props}/>,
```

Comment:
🤔 do we need props here? All other cases does not have any props for `KubeEventDetails`

Reply:
As I tried to explain with the comment on line 73 I left these two lines with props as a way to make sure that we don't break extension developers that use either of these styles. We have in the past so I think it would be good to have all three versions in our code.